nss-day-cohort-29/reactive-nutshell-copperden
# Ab frinedspart

# Description
Friends tab displays current user's friends. Add friend button goes to add friend page.

## Type of change

# Testing Instructions for Change Made
1. `git fetch --all`
2. `git checkout ab-frinedspart`
3. Run server
4. Verify .....

# Checklist:
